fixit/rules/sorted_attributes_rule.py

```diff
@@ -0,0 +1,117 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import List, Union
+
+import libcst as cst
+import libcst.matchers as m
+
+from fixit import CstLintRule, InvalidTestCase as Invalid, ValidTestCase as Valid
+
+
+LineType = Union[cst.BaseSmallStatement, cst.BaseStatement]
+
+
+class SortedAttributesRule(CstLintRule):
+    """
+    Ever wanted to sort a bunch of class attributes alphabetically?
+    Well now it's easy! Just add "@sorted-attributes" in the doc string of
+    a class definition and lint will automatically sort all attributes alphabetically.
+
+    Feel free to add other methods and such -- it should only affect class attributes.
+    """
+
+    INVALID = [
+        Invalid(
+            """
+            class MyUnsortedConstants:
+                \"\"\"
+                @sorted-attributes
+                \"\"\"
+                z = "hehehe"
+                B = 'aaa234'
+                A = 'zzz123'
+                cab = "foo bar"
+                Daaa = "banana"
+
+                @classmethod
+                def get_foo(cls) -> str:
+                    return "some random thing"
+           """,
+            expected_replacement="""
+            class MyUnsortedConstants:
+                \"\"\"
+                @sorted-attributes
+                \"\"\"
+                A = 'zzz123'
+                B = 'aaa234'
+                Daaa = "banana"
+                cab = "foo bar"
+                z = "hehehe"
+
+                @classmethod
+                def get_foo(cls) -> str:
+                    return "some random thing"
+           """,
+        )
+    ]
+    ONCALL_SHORTNAME = "ig_creation_backend"
```

Comment:
This is not needed.
```suggestion
```

Reply:
Nice catches 👍 . I'm pushing a new version that solves all the comments.